core/src/main/scala/org/apache/spark/ui/ToolTips.scala

```diff
@@ -62,6 +62,13 @@ private[spark] object ToolTips {
     """Time that the executor spent paused for Java garbage collection while the task was
        running."""
 
+  val PEAK_EXECUTION_MEMORY =
+    """Execution memory refers to the memory used by internal data structures created during
+       shuffles, aggregations and joins when Tungsten is enabled. The value of this accumulator
+       should be approximately the sum of the peak sizes across all such data structures created
+       in this task. For SQL jobs, this only tracks all unsafe operators, broadcast joins, and
+       external sort."""
```

Comment:
Its not even all of shuffle, though ...
I agree that "spark.shuffle.memoryFraction" is not good either, but it seems like now we've got another name that is also confusing and misleading.

Does this exactly correspond to what we *are* tracking w/ "spark.shuffle.memoryFraction"?  If so, we should probably highlight that in the tooltip ... and maybe even use it in the name since we're already stuck with that, for better or worse.

If this doesn't correspond to "spark.shuffle.memoryFraction", than I'm a little worried that its a bit of a random mix of different sources of memory usage.